com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -722,7 +722,7 @@ public override void Setup(ScriptableRenderContext context, ref RenderingData re
             }
 
             // Set the depth texture to the far Z if we do not have a depth prepass or copy depth
-            if (!requiresDepthPrepass && !requiresDepthCopyPass)
+            if (cameraData.renderType == CameraRenderType.Base && !requiresDepthPrepass && !requiresDepthCopyPass)
```

Comment:
Could you add to the comment above the reasoning behind this?

Reply:
added comment in e6049c62a06b9f1374cc18d24c8bf3e1a4bb16a2